composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -157,6 +164,27 @@ class KeywordInput extends React.Component {
 		this.props.onChange( event.target.value );
 	}
 
+	/**
+	 * Renders the input's label.
+	 *
+	 * @returns {ReactElement} The input label.
+	 */
+	renderLabel() {
+		const {
+			id,
+			label,
+			postLabelElement,
```

Comment:
Not immediately clear what `postLabelElement` especially because it refers to `post`? I can understand we're talking about a label, a field, and some element that is meant to be rendered here and it happens to be the Help link. Nothing that I can infer from the prop name `postLabelElement`.